com.unity.netcode.gameobjects/Editor/NetworkBehaviourEditor.cs

```diff
@@ -90,7 +90,8 @@ private void RenderNetworkVariable(int index)
 
             var behaviour = (NetworkBehaviour)target;
 
-            if (behaviour.NetworkManager != null && behaviour.NetworkManager.IsListening)
+            // Only server can MODIFY. So allow modification if network is either not running or we are server
+            if (behaviour.NetworkManager == null || !behaviour.NetworkManager.IsListening || behaviour.NetworkManager.IsServer)
```

Comment:
Actually NetworkBehaviourEditor is not a NetworkBehaviour, so you get an error here

ullReferenceException: Object reference not set to an instance of an object
Unity.Netcode.NetworkBehaviour.get_NetworkManager () (at /Users/matt.walsh/dev/com.unity.multiplayer.mlapi/com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs:174)
Unity.Netcode.Editor.NetworkBehaviourEditor.RenderNetworkVariableValueType[T] (System.Int32 index) (at /Users/matt.walsh/dev/com.unity.multiplayer.mlapi/com.unity.netcode.gameobjects/Editor/NetworkBehaviourEditor.cs:93)
System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <76cf9df0db0d4d0ca5e0b61b6e332823>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <76cf9df0db0d4d0ca5e0b61b6e332823>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <76cf9df0db0d4d0ca5e0b61b6e332823>:0)
Unity.Netcode.Editor.NetworkBehaviourEditor.RenderNetworkVariable (System.Int32 index) (at /Users/matt.walsh/dev/com.unity.multiplayer.mlapi/com.unity.netcode.gameobjects/Editor/NetworkBehaviourEditor.cs:73)
Unity.Netcode.Editor.NetworkBehaviourEditor.OnInspectorGUI () (at /Users/matt.walsh/dev/com.unity.multiplayer.mlapi/com.unity.netcode.gameobjects/Editor/NetworkBehaviourEditor.cs:175)
UnityEditor.UIElements.InspectorElement+<>c__DisplayClass59_0.<CreateIMGUIInspectorFromEditor>b__0 () (at /Users/bokken/buildslave/unity/build/ModuleOverrides/com.unity.ui/Editor/Inspector/InspectorElement.cs:636)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&) (at /Users/bokken/buildslave/unity/build/Modules/IMGUI/GUIUtility.cs:189)

Reply:
Do you have a NetworkedObject?